metron-extensions/metron-parser-extensions/metron-parser-bro-extension/metron-parser-bro/src/main/config/zookeeper/enrichments/bro.json

```diff
@@ -0,0 +1,17 @@
+{
+    "enrichment" : {
```

Comment:
I'm tending to lean towards being able to package up the whole thing on this, if only because it doesn't seem to have a lot of meaning to just pack up a parser without actually doing anything downstream.  As long as the downstream stuff is overridable (and since it's ZK stuff, it should be), I'm not terribly concerned if we provide semi-reasonable defaults (I mean, we are already, right?)

Otherwise, we're just shifting the burden of defining this stuff elsewhere, where it's less logically related, imo.